scripts/linters/pylint_extensions_test.py

```diff
@@ -3340,3 +3340,113 @@ def __init__(self, fake_arg):
 
         with checker_test_object.assertNoMessages():
             checker_test_object.checker.visit_classdef(metaclass_node)
+
+
+class DisallowHandlerWithoutSchemaTests(unittest.TestCase):
+
+    def test_non_schema_handlers_for_request_args_raise_error(self):
```

Comment:
Is this a non-schema handler? I am a bit confused by the test name. Perhaps `test_schema_handlers_without_request_args_raise_error` is what you meant?

Reply:
Done